src/main/java/pom/NaptolHomePage.java

```diff
@@ -0,0 +1,87 @@
+package pom;
+
+import java.util.List;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+
+public class NaptolHomePage {
+	@FindBy(xpath="//a[@id='login-panel-link']")private WebElement loginorRegister;
```

Comment:
Name should be in lower camel case.
Ex: loginOrRegister

Reply:
Done in camelcase.